solr/test-framework/src/java/org/apache/solr/embedded/JettySolrRunner.java

```diff
@@ -402,41 +384,31 @@ public synchronized void lifeCycleStarted(LifeCycle arg0) {
                   .setAttribute(SolrDispatchFilter.PROPERTIES_ATTRIBUTE, nodeProperties);
               root.getServletContext()
                   .setAttribute(SolrDispatchFilter.SOLRHOME_ATTRIBUTE, solrHome);
+
               SSLConfigurationsFactory.current().init(); // normally happens in jetty-ssl.xml
-              coreContainerProvider = new CoreContainerProvider();
-              coreContainerProvider.init(root.getServletContext());
+
               log.info("Jetty properties: {}", nodeProperties);
 
-              debugFilter =
-                  root.addFilter(DebugFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
-              extraFilters = new ArrayList<>();
-              for (Map.Entry<Class<? extends Filter>, String> entry :
-                  config.extraFilters.entrySet()) {
-                extraFilters.add(
-                    root.addFilter(
-                        entry.getKey(), entry.getValue(), EnumSet.of(DispatcherType.REQUEST)));
-              }
-
-              for (Map.Entry<ServletHolder, String> entry : config.extraServlets.entrySet()) {
-                root.addServlet(entry.getKey(), entry.getValue());
-              }
-              dispatchFilter = root.getServletHandler().newFilterHolder(Source.EMBEDDED);
-              dispatchFilter.setHeldClass(SolrDispatchFilter.class);
-              dispatchFilter.setInitParameter("excludePatterns", excludePatterns);
-              // Map dispatchFilter in same path as in web.xml
-              root.addFilter(dispatchFilter, "/*", EnumSet.of(DispatcherType.REQUEST));
-
-              synchronized (JettySolrRunner.this) {
-                waitOnSolr = true;
-                JettySolrRunner.this.notify();
-              }
-            }
-
-            @Override
-            public void lifeCycleFailure(LifeCycle arg0, Throwable arg1) {
-              System.clearProperty("hostPort");
+              super.contextInitialized(event);
             }
           });
+
+      debugFilter = root.addFilter(DebugFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
```

Comment:
No longer is this section in a LifeCycle.Listener.  No longer does it do JettySolrRunner.this.notify(); waitOnSolr is gone.